### PR TITLE
Feature |TRAINING | workflow-model

### PR DESCRIPTION
**Pull request by vishnu-aurx**

### CODE REVIEW

**Review by akashsen-aurx — APPROVED** [review]

File: core/src/main/java/com/aurx/core/servlets/MyWorkflowServlet.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This Servlet Hit The WorkFlow Model
```
Comment (abhinav-aggarwal-aem) [inline core/src/main/java/com/aurx/core/servlets/MyWorkflowServlet.java:23]: pls write a valid class description

File: core/src/main/java/com/aurx/core/servlets/MyWorkflowServlet.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ */
+@Component(service = Servlet.class, immediate = true, property = {
+    "sling.servlet.methods=GET",
+    "sling.servlet.paths=/bin/testworkflow",
+     "sling.servlet.selectors=myworkflow",
+     "sling.servlet.extensions=json"
+})
+public class MyWorkflowServlet extends SlingSafeMethodsServlet {
+
+  private static final Logger log = LoggerFactory.getLogger(MyWorkflowServlet.class);
```
Comment (abhinav-aggarwal-aem) [inline core/src/main/java/com/aurx/core/servlets/MyWorkflowServlet.java:33]: java docs

File: core/src/main/java/com/aurx/core/workflow/Branch1Workflow.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import com.adobe.granite.workflow.exec.WorkflowProcess;
+import com.adobe.granite.workflow.metadata.MetaDataMap;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component(service = WorkflowProcess.class,immediate = true,property = {
+    "process.label=vishnu branch1 workflow"
+})
+public class Branch1Workflow implements WorkflowProcess {
```
Comment (abhinav-aggarwal-aem) [inline core/src/main/java/com/aurx/core/workflow/Branch1Workflow.java:15]: java docs

File: core/src/main/java/com/aurx/core/workflow/MyWorkflowProcess.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import com.adobe.granite.workflow.exec.WorkflowProcess;
+import com.adobe.granite.workflow.metadata.MetaDataMap;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component(service = WorkflowProcess.class,immediate = true,property = {
+    "process.label=VishnuWorkflow"
+})
+public class MyWorkflowProcess implements WorkflowProcess{
```
Comment (abhinav-aggarwal-aem) [inline core/src/main/java/com/aurx/core/workflow/MyWorkflowProcess.java:15]: java docs

File: core/src/main/java/com/aurx/core/workflow/Branch2Workflow.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import com.adobe.granite.workflow.exec.WorkflowProcess;
+import com.adobe.granite.workflow.metadata.MetaDataMap;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component(service = WorkflowProcess.class,immediate = true,property = {
+    "process.label=vishnu branch 2 workflow"
+})
+public class Branch2Workflow implements WorkflowProcess {
```
Comment (abhinav-aggarwal-aem) [inline core/src/main/java/com/aurx/core/workflow/Branch2Workflow.java:15]: java docs

**Pull request merged**